DataFormats/interface/Muon.h

```diff
@@ -73,6 +112,8 @@ namespace cat {
     int numberOfMatchedStations_;
     int numberOfValidPixelHits_;
     int trackerLayersWithMeasurement_;
+
+    unsigned int selectors_;
```

Comment:
This introduces dataformat change. Increasing cattuple version name could be better.

Reply:
Did you mean cat91x or something like this? Yes, it will be more clear to increase version, but notice that only one group is using cattools for 2017 analysis: after finalizing tools and deleting old cattuples, tag for V9_4 will be sufficient, I guess.